packages/lesswrong/lib/collections/conversations/views.ts

```diff
@@ -25,9 +25,10 @@ Conversations.addDefaultView(function (terms: ConversationsViewTerms) {
 
 // notifications for the site moderation team
 Conversations.addView("moderatorConversations", function (terms: ConversationsViewTerms) {
+  const participantIds = terms.userId ? {participantIds: terms.userId} : {}
   const showArchivedFilter = terms.showArchive ? {} : {archivedByIds: {$ne: terms.userId}}
   return {
-    selector: {moderator: true, messageCount: {$gt: 0}, ...showArchivedFilter},
+    selector: {moderator: true, messageCount: {$gt: 0}, ...showArchivedFilter, ...participantIds},
```

Comment:
I'm betting this isn't supported by an index, and I'm also betting it don't matter because there aren't enough conversations. Oh, and it might *especially* not matter if the N in question is the number of *moderator* conversations.

Reply:
I went and added participantIds to the index but not 100% sure whether that was correct.